app/yarn.lock

```diff
@@ -331,6 +331,121 @@
   dependencies:
     "@types/hammerjs" "^2.0.36"
 
+"@esbuild/aix-ppc64@0.20.0":
```

Comment:
It looks that esbuild has to be linked here too, it is a dependency of eslintPlugin